lml/src/main/java/com/github/czyzby/lml/parser/impl/attribute/OnRightClickLmlAttribute.java

```diff
@@ -0,0 +1,56 @@
+package com.github.czyzby.lml.parser.impl.attribute;
+
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+import com.github.czyzby.lml.parser.LmlParser;
+import com.github.czyzby.lml.parser.action.ActorConsumer;
+import com.github.czyzby.lml.parser.tag.LmlAttribute;
+import com.github.czyzby.lml.parser.tag.LmlTag;
+
+/** Attaches a ClickListener to the the actor, invoking a chosen action upon right button clicking on actor. Expects an action ID.
+ * Assigned action method might have {@link Params} as parameter to retrieve some event related data.
+ * Mapped to "onRightClick" and "rightClick" attribute names.
+ *
+ * @author Metaphore */
+public class OnRightClickLmlAttribute implements LmlAttribute<Actor> {
+
+    @Override
+    public Class<Actor> getHandledType() {
+        return Actor.class;
+    }
+
+    @Override
+    public void process(final LmlParser parser, final LmlTag tag, final Actor actor, final String rawAttributeData) {
+        final ActorConsumer<?, Params> action = parser.parseAction(rawAttributeData, tmpParams);
+        if (action == null) {
+            parser.throwError("Could not find action for: " + rawAttributeData + " with actor: " + actor);
+        }
+        actor.addListener(new ClickListener(1) {
+            @Override
+            public void clicked(final InputEvent event, final float x, final float y) {
+                tmpParams.actor = actor;
+                tmpParams.x = x;
+                tmpParams.y = y;
+                tmpParams.stageX = event.getStageX();
+                tmpParams.stageY = event.getStageY();
+                action.consume(tmpParams);
+                tmpParams.reset();
+            }
+        });
+    }
+
+    private static Params tmpParams = new Params();
+    public static class Params {
```

Comment:
I think other listeners do not use a similar `Params` class - they expect actions consuming an actor instead. I understand that you might have needed that in your application, but I believe at this point consistency among all listeners is more important.